### ISSUE TITLE
Definition of prediction cost and correct prediction -- only labelled?

### ISSUE THREAD

**Issue by EvenStrangest** [rinuboney/ladder#15]
Shouldn't the definitions for prediction cost and correct prediction (see current definitions below) be changed to only include labelled examples?

```
pred_cost = -tf.reduce_mean(tf.reduce_sum(outputs*tf.log(y), 1))  # cost used for prediction
correct_prediction = tf.equal(tf.argmax(y, 1), tf.argmax(outputs, 1))  # no of correct predictions
```

**Comment by rinuboney** [comment]
`correct_prediction` is only used for computing `accuracy`. In this script, the `accuracy` operator is only used for computing the accuracy of the model over the whole test. There are no unlabeled examples in this case. If it is used in other cases, then yes it should be computed only for labeled examples.
